src/elements/hoverboard-icons.html

```diff
@@ -49,7 +49,15 @@
.297 0.414l2.344 0.55c0.189 0.044 0.381-0.058 0.45-0.239 0.326-0.852 1.641-0.541 1.554 0.365-0.019 0.193 0.108 0.37 0.297 0.414l2.344 0.55c0.186 0.044 0.382-0.060 0.45-0.239 0.118-0.309 0.42-0.517 0.751-0.517 0.171 0 0.317 0.073 0.477-0.026 0.087-0.054 0.149-0.141 0.173-0.241l0.509-2.169c0 0 1.091 0.756 1.092 0.756 0.154 0.107 0.376 0.085 0.506-0.057 0.152-0.166 0.369-0.261 0.595-0.261 0.164 0 0.322 0.050 0.457 0.143 0.176 0.122 0.417 0.078 0.539-0.098l9.919-14.328c0.122-0.176 0.078-0.417-0.098-0.539zM10.559 16.884c0 0-0.583 2.486-0.583 2.486l-0.561 2.392c-0.523 0.009-1.006 0.281-1.289 0.708l-1.761-0.413c-0.081-0.642-0.555-1.189-1.206-1.342-0.636-0.149-1.315 0.122-1.677 0.666l-1.76-0.413c-0.065-0.514-0.382-0.968-0.841-1.207l3.235-13.788 0.377 0.088c0.030 0.007 0.059 0.010 0.089 0.010 0.176 0 0.335-0.12 0.376-0.299 0.049-0.208-0.080-0.416-0.288-0.465l-0.377-0.088 0.414-1.764c0.523-0.009 1.006-0.281 1.289-0.708l1.761 0.413c0.081 0.642 0.555 1.19 1.206 1.342 0.636 0.149 1.315-0.122 1.677-0.666l1.761 0.413c0.005 0.001 0.016 0.102 0.018 0.113 0.023 0.114 0.060 0.229 0.108 0.336 0.146 0.322 0.399 0.595 0.714 0.758l-0.414 1.764-0.377-0.088c-0.208-0.049-0.416 0.080-0.465 0.288s0.080 0.416 0.288 0.465l0.377 0.088-0.616 2.623c0 0-0.067 0.287-0.067 0.287l-0.452 1.926c-0-0-0.956 4.073-0.956 4.074zM21.847 7.583l-0.318-0.22c-0.176-0.122-0.417-0.078-0.539 0.098s-0.078 0.417 0.098 0.539l0.318 0.22-8.061 11.644c-0.176-0.066-0.362-0.1-0.552-0.1-0.327 0-0.644 0.102-0.907 0.286l-1.096-0.759 0.393-1.676 1.712 1.185c0.174 0.12 0.418 0.076 0.539-0.098l4.977-7.189c0.122-0.176 0.078-0.417-0.098-0.539s-0.417-0.078-0.539 0.098l-4.757 6.871-1.644-1.138 0.755-3.216 1.359-0.476 0.881 1.156c0.075 0.098 0.189 0.153 0.308 0.153 0.043 0 0.086-0.007 0.128-0.022 0.158-0.055 0.263-0.206 0.259-0.374l-0.033-1.453 1.371-0.48c0.158-0.055 0.263-0.206 0.259-0.374s-0.115-0.314-0.276-0.362l-1.392-0.418-0.033-1.453c-0.004-0.168-0.115-0.314-0.276-0.362s-0.334 0.013-0.429 0.15l-0.827 1.195-0.53-0.159 0.996-4.246 0.488-0.705 4.967 3.439-0.674 0.973c-0.122 0.176-0.078 0.417 0.098 0.539s0.417 0.078 0.539-0.098c0 0 0.891-1.287 0.894-1.291 0.12-0.173 0.074-0.419-0.098-0.539 0 0-5.594-3.873-5.604-3.88-0.174-0.12-0.418-0.076-0.539 0.098l-0.168 0.243c-0.005-0.001-0.009-0.003-0.014-0.004-0.191-0.045-0.36-0.16-0.473-0.32-0.001-0.001-0.002-0.002-0.002-0.004l0.89-1.286c0 0 0.318 0.22 0.318 0.22 0.169 0.117 0.417 0.078 0.539-0.098s0.078-0.417-0.098-0.539l-0.318-0.22 1.031-1.49c0.176 0.066 0.362 0.1 0.552 0.1 0.327 0 0.644-0.102 0.907-0.286l1.487 1.029c-0.16 0.627 0.081 1.31 0.631 1.691 0.538 0.372 1.269 0.37 1.805-0.005l1.487 1.029c-0.128 0.502 0.001 1.041 0.34 1.431l-1.032 1.49zM14.262 12.851c0 0-0.329-0.432-0.329-0.432-0.1-0.132-0.28-0.185-0.436-0.131l-0.513 0.18c0.169-0.244 0.535-0.592 0.299-0.902l-0.329-0.432c0 0 0.52 0.156 0.52 0.156 0.363 0.109 0.572-0.356 0.739-0.597 0 0 0.012 0.542 0.012 0.543 0.009 0.378 0.518 0.434 0.796 0.518l-0.513 0.18c-0.158 0.055-0.263 0.207-0.259 0.374l0.012 0.543z"></path>
```

Comment:
Everything is correct. Look at image it has a lot of paths + it's GitHub failed to highlight last line, look at code ;)